networkdb/broadcast.go

```diff
@@ -17,8 +17,9 @@ type networkEventMessage struct {
 }
 
 func (m *networkEventMessage) Invalidates(other memberlist.Broadcast) bool {
-	otherm := other.(*networkEventMessage)
-	return m.id == otherm.id && m.node == otherm.node
+	//otherm := other.(*networkEventMessage)
+	//return m.id == otherm.id && m.node == otherm.node
+	return false
```

Comment:
I would try to keep this, also with the monotonic increasing epoch this should not create issues